### ISSUE TITLE
Chronologisch absteigend

### ISSUE THREAD

**Issue by annekauf** [StabiBerlin/sbb-relevance-test#11]
Bei allen Suchanfragen: Trefferliste soll nach Erscheinungsjahr absteigend gerankt sein, unabhängig von der Medienart.
Cutoff nach x Jahren (tbd; Sonderfall Zsn für Cutoff?)

Ausnahme: Suche nach Jahr

**Comment by duncdrum** [comment]
ok nach einigen tests: Datumsfunktionen erwarten im Index ein Datumsfeld. Wir haben jedoch ein Textfelder für `publishDate` und `publishDateSort` . Das limitiert was wir via ranking config gerade ziehen können.

**Comment by annekauf** [comment]
indeed... das war's dann mit chronologischem Ranking, dann geht nur Boost über letzte x Ersch.jahre.

**Comment by annekauf** [comment]
Lese gerade die Lit.Hinweise von Anne Christensen und finde hier
[](https://www.bib-bvb.de/documents/10967/9184676/Solr-Ranking+6-2018.pdf/3aaaa96c-8716-447a-8163-584f3ddf73f3)

diese Funktion für Chrono-Ranking:
bf = recip(ms(NOW/YEAR,freshness),3.16e-11,1,1)^10.0

Could this work for us?

**Comment by duncdrum** [comment]
see #59 `NOW/YEAR,` returns dates but we are not dealing with dates
